botocore/signers.py

```diff
@@ -196,6 +208,65 @@ def generate_presigned_url(self, request_dict, expires_in=3600,
         return request.url
 
 
+class CloudFrontSigner(object):
```

Comment:
But I thought "pre-signed" is a naming convention only for S3. For example, in [S3 document, there are 19 occurrence of "pre-signed" and they don't use "signed"](http://docs.aws.amazon.com/AmazonS3/latest/dev/PresignedUrlUploadObject.html). On the contrary, [in CloudFront document, the naming convention is always saying "signed", never "pre-signed"](http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-creating-signed-url-custom-policy.html). So, shall we keep the current "CloudFrontSigner" as-is?